Formula/root.rb

```diff
@@ -73,7 +83,7 @@ def install
       -Dfortran=ON
       -Dgdml=ON
       -Dgnuinstall=ON
-      -Dimt=ON
+      -Dtbb=ON
```

Comment:
`-Dtbb=ON` does not exist. `-Dimt=ON` is really the correct option.

Reply:
Okay, fixed this for next time I push. Thanks!